.travis.yml

```diff
@@ -6,14 +6,7 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-## PHP versions we test against
-php:
-  - 7.0
-  - 7.1
-  - nightly
-  - hhvm
-
-## Build matrix for lowest and highest possible targets
```

Comment:
I think I was seeing something other than what I was thinking I was seeing. In any case, I have simplified things to get the same build you wanted.